### PR TITLE
Supplier add component

### PR DESCRIPTION
**Pull request by Muh-Baharuddin**
@sidratul please review my code.

### CODE REVIEW

File: src/containers/Supplier/Supplier.tsx [hunk 1]
```diff
@@ -1,15 +1,61 @@
...
         <div className="card-header">
-          <AddComp />
+          <AddComp refreshSupplier={refreshSupplier}/>
         </div>
-        <TableSupplier />
+        <TableSupplier 
+          data={data}
+          queryParams={queryParams}
+          setQueryParams={setQueryParams}
+          refreshSupplier={refreshSupplier}
```
Comment (sidratul) [inline src/containers/Supplier/Supplier.tsx:57]: create as context

Comment (Muh-Baharuddin) [reply src/containers/Supplier/Supplier.tsx]: done

File: src/containers/Supplier/Supplier.tsx [hunk 2]
```diff
@@ -1,16 +1,93 @@
...
+    total: 0,
+  });
+
+  const [queryParams, setQueryParams] = useState<QueryParamsType>({
+    keywords: '',
+    orderBy: 'nama',
+    orderType: '',
+    page: 1,
+    limit: 10,
+  })
```
Comment (sidratul) [inline src/containers/Supplier/Supplier.tsx:61]: its defined in defaultState so use it. 
`useState<QueryParamsType>(defaultState.queryParams)`

Comment (Muh-Baharuddin) [reply src/containers/Supplier/Supplier.tsx]: done, thanks for the info

**Review by sidratul — APPROVED** [review]

**Pull request merged**